### ISSUE TITLE
Option to disable Default Template Layout

### ISSUE THREAD

**Issue by ghost** [go-aah/aah#51]
Not all templates use Layout, can you cancel DefaultTmplLayout?
```
if ess.IsStrEmpty(htmlRdr.Layout) {
	htmlRdr.Layout = appDefaultTmplLayout
}
```

**Comment by jeevatkm** [comment]
One of the notable feature in aah framework is multi layout and its flexibility. I understand your application need.

I will provide a configuration option to disable `DefaultTmplLayout`?

**Comment by jeevatkm** [comment]
For now create an empty layout file, something like `nolayout.html` and call 
```go
HTMLlf("nolayout.html", "/path/to/tmplfile.html", data)
```

**Comment by LeSou** [comment]
thank you

**Comment by LeSou** [comment]
@jeevatkm 
`default_layout = false`
View not found: views\pages\frontend\index\index.html

[https://github.com/go-aah/aah/blob/10f79dfc4e788a998c26d0c2abf0b95945621bab/view.go#L183](url)

[https://github.com/go-aah/view/blob/v0-unstable/go_engine.go#L70](url)

**Comment by jeevatkm** [comment]
Did you manually apply commit changes into your local codebase?

**Comment by LeSou** [comment]
I have not processed yet

**Comment by jeevatkm** [comment]
@LeSou this config option to be release in `v0.6`. If you would like to try now, please do these steps.

Patch files created from commit sha.
 * [0001-50-tmpl-auto-resolve-fix-and-enhancement.txt](https://github.com/go-aah/aah/files/1033252/0001-50-tmpl-auto-resolve-fix-and-enhancement.txt)
 * [0001-51-config-option-to-disable-default-tmpl-layout.txt](https://github.com/go-aah/aah/files/1033253/0001-51-config-option-to-disable-default-tmpl-layout.txt)

```bash
# go to directory
cd $GOPATH/src/aahframework.org/aah.v0

# run this
git reset --hard HEAD
patch -p1 < 0001-50-tmpl-auto-resolve-fix-and-enhancement.txt
patch -p1 < 0001-51-config-option-to-disable-default-tmpl-layout.txt
```

Now test it and let me know.

**Comment by LeSou** [comment]
hello,not solved
$ patch -p1 < 0001-50-tmpl-auto-resolve-fix-and-enhancement.txt
patching file aah_test.go
patching file context.go
patching file context_test.go
patching file controller.go
patching file engine_test.go
patching file middleware.go
patching file util.go
patching file view.go
patching file view_test.go

$ patch -p1 < 0001-51-config-option-to-disable-default-tmpl-layout.txt
patching file view.go
patching file view_test.go

aah.conf

`default_layout = false`

2017-05-27 11:10:09.685 ERROR template not found: views\pages\frontend\index\index.html


`default_layout = true`  Normal use。

**Comment by jeevatkm** [comment]
I will check and get back to you, thanks.

**Comment by jeevatkm** [comment]
@LeSou Can you apply this patch also and test it?

[view-nolayout.txt](https://github.com/go-aah/aah/files/1033307/view-nolayout.txt)

```bash
# go to directory
cd $GOPATH/src/aahframework.org/view.v0

# run this
git apply --ignore-space-change --ignore-whitespace view-nolayout.txt
```

**Comment by LeSou** [comment]
solve, thank you

**Comment by jeevatkm** [comment]
I'm glad to hear that, thanks.

BTW, I have a question. when you say no layout? do you mean you don't have header, footer etc. for the certain pages?

**Comment by LeSou** [comment]
Yes, i am using **angular2**

**Comment by jeevatkm** [comment]
Implemented, to be released with `v0.6`.
